projects/sandbox/pyproject.toml

```diff
@@ -98,3 +100,20 @@ kernel_length = "${base.kernel_length}"
 waveform_duration = "${base.waveform_duration}"
 datadir = "${base.datadir}"
 logdir = "${base.logdir}"
+
+[tool.typeo.scripts.sample_testing_set]
+ifos = "${base.ifos}"
+kernel_length = "${base.kernel_length}"
+sample_rate = "${base.sample_rate}"
+fduration = "${base.fduration}"
+trigger_distance = 0.0
+model-state-path = "${base.basedir}/training/weights.pt"
+inference_params = ["frequency", "quality", "hrss", "dec", "psi", "phi"]
```

Comment:
Since `inference_params` is used in the `train` project as well, could we upgrade this to a `base` variable?

Reply:
Same with `trigger_distance`